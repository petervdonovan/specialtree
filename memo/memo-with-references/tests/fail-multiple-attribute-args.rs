use memo_with_references::memo;

#[memo('a, 'b)]
fn test_multiple_lifetimes() {}

fn main() {}
