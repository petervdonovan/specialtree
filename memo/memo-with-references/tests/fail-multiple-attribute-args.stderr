error: unexpected token
 --> tests/fail-multiple-attribute-args.rs:3:10
  |
3 | #[memo('a, 'b)]
  |          ^
